modules/integration/tests-common/admin-clients/src/main/java/org/wso2/mb/integration/common/clients/AndesClient.java

```diff
@@ -1,589 +1,317 @@
-
-            if (operation.equals("printMissing")) {
-
-                numberOfMessagesSentAsString = numberOfMessagesExpectedForAnalysis;
-                numberOfMessagesSent = Integer.parseInt(numberOfMessagesSentAsString);
-            }
-
-            //print input information
-            log.info("mode==>" + mode);
-            log.info("operation==>" + operation);
-
-            AndesClientOutputParser andesClientOutputParser = new AndesClientOutputParser(filePathToWriteReceivedMessages);
-
-            if (operation.equals("printMessages")) {
-                andesClientOutputParser.printMessagesMap();
-            } else if (operation.equals("printDuplicates")) {
-                andesClientOutputParser.printDuplicateMessages();
-            } else if (operation.equals("printMissing")) {
-                andesClientOutputParser.printMissingMessages(numberOfMessagesSent);
-            } else if (operation.equals("printSorted")) {
-                andesClientOutputParser.printMessagesSorted();
-            } else if (operation.equals("checkOrder")) {
-                log.info("MESSAGE ORDER PRESERVED: " + andesClientOutputParser.checkIfMessagesAreInOrder());
-            } else if (operation.equals("clearFile")) {
-                andesClientOutputParser.clearFile();
-            } else {
-                log.info("analyse operation not found...");
             }
         } else {
-            log.info("ERROR: Unknown mode of operation");
+            throw new AndesClientException("The amount of subscribers cannot be less than 1. " +
+                                           "Value entered is " + Integer.toString(numberOfThreads));
         }
     }
 
-    private int browseQueue(String host, String port, String userName, String password, String destination, int printNumberOfMessagesPer, boolean isToPrintMessage) {
-        int messageCount = 0;
-        //create a browser - one threaded app blocking
-        //make an enumeration and get the count
-        QueueMessageBrowser queueMessageBrowser = new QueueMessageBrowser(host, port, userName, password, destination, printNumberOfMessagesPer, isToPrintMessage);
-        messageCount = queueMessageBrowser.getMessageCount();
-        return messageCount;
-    }
-
-
-    //******************************************************************************************************************
-
-    public void shutDownClient() {
-
-        if (mode.equals("send")) {
-
-            for (QueueMessageSender qSender : queueMessageSenders) {
-                qSender.stopSending();
+    /**
+     * Starts up the consumer(s) or publisher(s) to consume or publish messages.
+     *
+     * @throws JMSException
+     * @throws IOException
+     */
+    public void startClient() throws AndesClientException, JMSException, IOException {
+        for (AndesJMSConsumer consumer : consumers) {
+            consumer.startClient();
+            if (this.startDelay > 0L) {
```

Comment:
In this if(this.startDelay > 0L) will take more time than if(boolean) since in the first one a integer comparison is happening. There is no different if this is happening only once. But since this is running in a loop, the same will happen again and again. Therefore the time this if condition takes overall can be reduced by saving it in a variable.